visidata/loaders/postgres.py

```diff
@@ -98,7 +95,7 @@ def reload(self):
             r = cur.fetchone()
             if r:
                 self.addRow(r)
-            self.columns = cursorToColumns(cur)
+            cursorToColumns(cur, self)
```

Comment:
sure, that'll work.  Whenever we pass 'self' into a function we should consider if it should be a member function (i.e. called as `self.cursorToColumns(cur))`.  Probably in this case it would be neater but since this is isolated in an optional loader I'm less picky about style things like that.

Reply:
I was considering just removing the function, as it is basically two lines and doesn't really remove complexity from the sheets. Just didn't feel comfortable to do a whole refactor of the code there yet.